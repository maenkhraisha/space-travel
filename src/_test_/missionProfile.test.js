import React from 'react';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import MissionProfile from '../Components/missionProfile';
import store from '../redux/configureStore';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <MissionProfile />
      </Provider>,
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
