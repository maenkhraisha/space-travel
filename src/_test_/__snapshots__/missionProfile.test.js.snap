// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="reserve-container-top"
>
  <h3>
    My Missions
  </h3>
  <table
    className="table table-bordered"
  >
    <p>
      No Reserved Missions
    </p>
  </table>
</div>
`;
